components/Learn.js

```diff
@@ -4,23 +4,84 @@ import Topic from "../components/Topic";
 
 const Learn = () => {
 	return (
-		<div className='font-lexend text-lg'>
-			{" "}
-			<div className='flex justify-center items-center font-lexend font-bold text-4xl mb-4'>
+		<div className='w-full font-lexend text-4xl'>
+			<div className='flex justify-center items-center font-lexend font-bold text-6xl mb-2'>
 				Learn
 			</div>
+			<div className='py-10'>
```

Comment:
You can move the div and just apply the padding on the Row, it is a large value so make sure to test it out on different screen sizes